config.plist/alder-lake.md

```diff
@@ -0,0 +1,793 @@
+| Target | 67 |
+
+:::
+
+::: details More in-depth Info
+
+* **AppleDebug**: YES
+  * Enables boot.efi logging, useful for debugging. Note this is only supported on 10.15.4 and newer
+* **ApplePanic**: YES
+  * Attempts to log kernel panics to disk
+* **DisableWatchDog**: YES
+  * Disables the UEFI watchdog, can help with early boot issues
+* **DisplayLevel**: `2147483650`
+  * Shows even more debug information, requires debug version of OpenCore
+* **SerialInit**: NO
+  * Needed for setting up serial output with OpenCore
+* **SysReport**: NO
+  * Helpful for debugging such as dumping ACPI tables
+  * Note that this is limited to DEBUG versions of OpenCore
+* **Target**: `67`
+  * Shows more debug information, requires debug version of OpenCore
+
+These values are based of those calculated in [OpenCore debugging](../troubleshooting/debug.md)
+
+:::
+
+### Security
+
+::: tip Info
+
+Security is pretty self-explanatory, **do not skip**. We'll be changing the following:
+
+| Quirk | Enabled | Comment |
+| :--- | :--- | :--- |
+| AllowNvramReset | YES | |
+| AllowSetDefault | YES | |
+| BlacklistAppleUpdate | YES | |
+| ScanPolicy | 0 | |
+| SecureBootModel | Default | Leave this as `Default` if running macOS Big Sur or newer. The next page goes into more detail about this setting. |
+| Vault | Optional | This is a word, it is not optional to omit this setting. You will regret it if you don't set it to Optional, note that it is case-sensitive. |
+
+:::
+
+::: details More in-depth Info
+
+* **AllowNvramReset**: YES
+  * Allows for NVRAM reset both in the boot picker and when pressing `Cmd+Opt+P+R`
+* **AllowSetDefault**: YES
+  * Allow `CTRL+Enter` and `CTRL+Index` to set default boot device in the picker
+* **ApECID**: 0
+  * Used for netting personalized secure-boot identifiers, currently this quirk is unreliable due to a bug in the macOS installer so we highly encourage you to leave this as default.
+* **AuthRestart**: NO
+  * Enables Authenticated restart for FileVault 2 so password is not required on reboot. Can be considered a security risk so optional
+* **BlacklistAppleUpdate**: YES
+  * Used for blocking firmware updates, used as extra level of protection as macOS Big Sur no longer uses `run-efi-updater` variable
+
+* **DmgLoading**: Signed
+  * Ensures only signed DMGs load
+* **ExposeSensitiveData**: `6`
+  * Shows more debug information, requires debug version of OpenCore
+* **Vault**: `Optional`
+  * We won't be dealing vaulting so we can ignore, **you won't boot with this set to Secure**
+  * This is a word, it is not optional to omit this setting. You will regret it if you don't set it to `Optional`, note that it is case-sensitive
+* **ScanPolicy**: `0`
+  * `0` allows you to see all drives available, please refer to [Security](https://dortania.github.io/OpenCore-Post-Install/universal/security.html) section for further details. **Will not boot USB devices with this set to default**
+* **SecureBootModel**: Disabled
```

Comment:
Should probably be left as Default.

Reply:
Missed this one somehow as well. I'll update the Rocket Lake PR shortly. Resolved in f8ee8fc.